third_party/prometheus-nats-exporter-docker/amd64/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Golang binary building stage
-FROM golang:1.20.2
+FROM golang:1.21.6
```

Comment:
```suggestion
FROM golang:1.21
```

Reply:
It's hard to track down possible issues if the image is not static.

Reply:
Done